app/Http/Controllers/PrintController.php

```diff
@@ -48,6 +48,7 @@ public function print(Request $request) {
             return back()->withErros($validator)->withInput();
         }
         // Only calculating the values here to see how many pages can be covered free of charge.
+        // TODO: if possible, use allActiveFreePages()
```

Comment:
The problem is that the part below also collects the ones with the earliest deadline, so we cannot really use this here.